akka-cluster/src/test/scala/akka/cluster/DowningProviderSpec.scala

```diff
@@ -39,7 +39,7 @@ class DowningProviderSpec extends WordSpec with Matchers {
 
   val baseConf = ConfigFactory.parseString("""
       akka {
-        loglevel = WARNING
+        loglevel = INFO
```

Comment:
Do we want this?

Reply:
We use INFO on many/most cluster tests, makes it easier to see what's going on when failing. Should use log capturing, but that's for another day.